src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -194,6 +194,12 @@ class CodeBlock private constructor(
       }
 
       while (p < format.length) {
+        if (format[p] in setOf('⇥', '⇤', '«', '»')) {
```

Comment:
How about checking for these symbols in `isNoArgPlaceholder()`?

Reply:
There are a handful of places that use `isNoArgPlaceholder()` assuming that it's prefixed with a `%`. I'll rework this a bit (maybe a new method to check for single character argument placeholders)